default/stringtables/en.txt

```diff
@@ -6015,6 +6030,12 @@ The focus of a planet may grant it bonuses or enable effects. A small penalty ap
 
 A planet will default to the inhabiting species' preferred focus if it has one, otherwise it will default to [[metertype METER_INDUSTRY]].'''
 
+DISTORTION_FOCUS_TITLE
+Spatial Distortion Focus
+
+DISTORTION_FOCUS_TEXT
+Available after construction the [[BLD_SPATIAL_DISTORT_GEN]], the properties of each starlane leading to the planet can be manipulated to slow down incoming enemy ships, retarding their progress by 40uu per turn. Slow ships may even be forced back to their origin system.
```

Comment:
"origin" might be misleading... maybe "to the system at the other end of the starlane"? Kinda wordy, though...